rubocop.yml

```diff
@@ -0,0 +1,39 @@
+# Defaults: https://github.com/bbatsov/rubocop/blob/master/config/default.yml
+
+# Accept both [:a, :b, :c] and %i[a b c]
+Style/SymbolArray:
+  Enabled: false
+
+Metrics/LineLength:
+  Max: 120
+
+Layout/MultilineMethodCallIndentation:
+  EnforcedStyle: indented_relative_to_receiver
+
+Layout/IndentFirstHashElement:
+  EnforcedStyle: consistent
+
+Metrics/AbcSize:
+  Max: 15
```

Comment:
This could possibly be too small for a major part of our codebase, though.

Reply:
This is from the default value of Rubocop. I want to keep it for new projects, but of course we can always override this value for legacy projects :)